packages/wrangler/src/cfetch/index.ts

```diff
@@ -44,6 +45,43 @@ export async function fetchResult<ResponseType>(
 	}
 }
 
+/**
+ * Make a fetch request, and extract the `result` from the JSON response.
+ */
+export async function fetchScriptContent(
+	resource: string,
+	init: RequestInit = {},
+	queryParams?: URLSearchParams,
+	abortSignal?: AbortSignal
+): Promise<string> {
+	const response = await performApiFetch(
+		resource,
+		init,
+		queryParams,
+		abortSignal
+	);
+
+	logger.debug(
+		"-- START CF API RESPONSE:",
+		response.statusText,
+		response.status
+	);
+
+	logger.debug("HEADERS:", { ...response.headers });
+	// logger.debug("RESPONSE:", text);
+	logger.debug("-- END CF API RESPONSE");
```

Comment:
I feel like this should almost be inside `performApiFetch`, but I understand not wanting to put code somewhere your PR doesn't directly modify. I think this is fine for now. At some point we'll want to probably do a big logging fixup pass or something.